src/main/scala/org/broadinstitute/dsde/agora/server/model/AgoraEntity.scala

```diff
@@ -20,6 +20,16 @@ object AgoraEntity {
       case HttpEntity.Empty ⇒ new AgoraEntity()
     }
 
+  implicit val SeqAgoraEntityUnmarshaller =
+    Unmarshaller[Seq[AgoraEntity]](`application/json`) {
+      case HttpEntity.NonEmpty(contentType, data) => {
+        grater[AgoraEntity].fromJSONArray(data.asString)
+      }
+      case HttpEntity.Empty => {
+        Seq(new AgoraEntity())
```

Comment:
We may want to return an empty Seq... additionally I'm not sure returning new AgoraEntity above is a good idea either. Any thoughts?